task5.py

```diff
@@ -0,0 +1,39 @@
+'''
+=>You are given a list of numbers and your task is to find out the pairs that have sum equals n (provided during input).
+ No duplicate pair or similar pair should be in the output.  For example, if sum=12,
+o [4, 8] can’t come twice in the output.
+o Also, consider [4, 8] and [8, 4] as similar, so only [4, 8] have to come, not
+both.
+ Input:
+o Numbers = [9, 4, 8, 10, 2, 4, 8, 3, 14, 4, 8] o n = 12
+ Output:
+o [[3, 9], [4, 8], [2, 10]]
+See here, each pair addition is equals to 12 i.e. (3+9=12, 4+8=12, 2+10=12)
+'''
+
+
+from itertools import combinations
+#list
+numbers = [9, 4, 8, 10, 2, 4, 8, 3, 14, 4, 8]
+
+#combinations of list
+list_of_list=[]
+pairs = list(combinations(numbers,2))
+
+#remove duplicat values in combinations of list
+list_of_tuple_uniq = list(set(tuple(sorted(sub)) for sub in pairs))
```

Comment:
There is no need of this type of conversion. Please remove it and apply some better logic. This increases a time complexity if we have 100000 numbers.